src/app/navigation/navigation.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-navigation',
+  templateUrl: './navigation.component.html',
+  styleUrls: ['./navigation.component.scss']
+})
+export class NavigationComponent implements OnInit {
+
+  constructor() { }
```

Comment:
If you use a scanner like [Snyk](https://snyk.io/) or [Sonarqube] it will report issues about empty `ctor` and `ngOnInit`. The project **shouldn't** contain code that is not used or do anything. This practice makes your project more readable and easier to understand.